frontend/src/views/PresentationRequestDetails.vue

```diff
@@ -24,19 +24,19 @@
         <v-divider></v-divider>
       </v-card-text>
       <v-layout align-end justify-end>
-        <v-btn
+        <v-bpa-button
           color="secondary"
           text
```

Comment:
BPAButtons don't add in the 'text' attribute, that will get picked up by the global ux config and handled by the BPAButton component.